tests/Etsy/methods.json

```diff
@@ -9,6 +9,16 @@
     "visibility": "public",
```

Comment:
This is an update of the methods.json? Is it exactly the same json from Etsy's method?